### ISSUE TITLE
cannot switch to tagbar window when Goyo is enabled

### ISSUE THREAD

**Issue by gauteh** [junegunn/goyo.vim#124]
Hi, Goyo is great. However, I can't swap to the tagbar window when I have Goyo running. This is very useful for latex files.

**Comment by junegunn** [comment]
Yeah, Goyo tries hard to confine the cursor inside the central window to keep the layout intact. A workaround is to skip over the padding window with <kbd>2</kbd><kbd>CTRL-W</kbd><kbd>l</kbd> or <kbd>CTRL-W</kbd><kbd>2</kbd><kbd>l</kbd>. Or you can your mouse :)

I currently don't have an idea to make Goyo smarter so the trick is not required. I'll leave the issue open and think about it when I get some time.

**Comment by gauteh** [comment]
That probably works. For some reason I cant move the mousepointer out of
the goyo window, it just jumps back.

Den onsdag 31. august 2016 skrev Junegunn Choi <notifications@github.com>
følgende:
